lesson2/README.md

```diff
@@ -1,14 +1,102 @@
 # Lesson 2 - Android Basics
-
+### Your first Android application!
 Today in class we'll walk you through making your first Android application. Found in the lesson2 folder is a copy of the application that we will create in class.
 
-## [Activities](https://developer.android.com/guide/components/activities.html) and [Fragments](https://developer.android.com/guide/components/fragments.html)
+## Create a Blank App
+To begin with, we'll create a default app and walk you through the different components.
 
-more info
+1. Create a new project in Android Studio
+2. Name it something akin to "Lesson2HW" and save it in your forked repo
+3. Click "Basic Activity"
+4. Check "Use a Fragment"
+5. Click done
+6. Click the little green play arrow and run your app
 
+Great, you just created your very first app!
 
-## Extra Resources
+## Explore Around
+Android Studio just created a bunch of files and boilerplate code to get you started. Before we write any code, let's explore the different files that were just created.
+
+There are generally three different types of files that you'll need to know about in Android Studio:
+
+1. **Java files** are where your backend logic goes.
```

Comment:
Yeah maybe, I thought freshman won't really know that though.  I can change it to just "logic"